src/helpers/helper.js

```diff
@@ -8,6 +8,13 @@ function getJsonFromXMLFile(filePath) {
   return parser.parse(xml, { arrayMode: true, ignoreAttributes: false, parseAttributeValue: true });
 }
 
+function getJsonFromFile(filePath) {
```

Comment:
we can directly read a json file using `require` statement. Anyway we can keep it for now.

Reply:
Updated the PR to require the json instead of fs.readSync